help/destinations/catalog/personalization/algolia-user-segments.md

```diff
@@ -0,0 +1,147 @@
+
+| Field | Description |
+| --- | --- |
+| [!UICONTROL Name] | Fill in the preferred name for this destination. |
+| [!UICONTROL Description] | Short explanation of the destination's purpose. |
+| [!UICONTROL Region] | The options are **US** or **EU**. Select the region where the customer data is stored. |
+
+![Account details](../../assets/catalog/personalization/algolia-user-segments/account.png)
+
+### Existing account {#existing-account}
+
+Accounts already defined using the [!DNL Algolia User Segments] destination appear in a list pop-up. When selected, you can see details on the account in the right rail. View the example from the UI, when you navigate to **[!UICONTROL Destinations]** > **[!UICONTROL Accounts]**:
+
+![Existing Account](../../assets/catalog/personalization/algolia-user-segments/existing-account.png)
+
+### Enable alerts {#enable-alerts}
+
+You can enable alerts to receive notifications on the status of the dataflow to your destination. Select an alert from the list to subscribe to receive notifications on the status of your dataflow. For more information on alerts, see the guide on [subscribing to destinations alerts using the UI](../../ui/alerts.md).
+
+When you are finished providing details for your destination connection, select **[!UICONTROL Next]**.
+
+## Data usage and governance {#data-usage-governance}
+
+All [!DNL Adobe Experience Platform] destinations are compliant with data usage policies when handling your data. For detailed information on how [!DNL Adobe Experience Platform] enforces data governance, read the [Data Governance overview](https://experienceleague.adobe.com/docs/experience-platform/data-governance/home.html).
+
+![Governance](../../assets/catalog/personalization/algolia-user-segments/governance.png)
+## Activate audiences to this destination {#activate}
+
+>[!IMPORTANT]
+> 
+>To activate data, you need the **[!UICONTROL View Destinations]**, **[!UICONTROL Activate Destinations]**, **[!UICONTROL View Profiles]**, and **[!UICONTROL View Segments]** [access control permissions](/help/access-control/home.md#permissions). Read the [access control overview](/help/access-control/ui/overview.md) or contact your product administrator to obtain the required permissions.
+
+Read [Activate audiences to edge personalization destinations](../../ui/activate-edge-personalization-destinations.md) for instructions on activating audiences to this destination.
+
+![Activate](../../assets/catalog/personalization/algolia-user-segments/audiences.png)
+
+### Mapping considerations {#mapping-considerations}
+
+The table below provides recommended mappings for target fields on your destination side that are used by Algolia.
+
+| Target Field | Source Description |
+|--------------|-------------------------------------------------------------|
+| userId | The `IdentityMap: AlgoliaUserToken` value in Experience Platform. |
```

Comment:
Use the first column in the table to describe what source identity to use and the second column for the target one.